File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        boolean flag = true;
        for (int i = 0; i < s.length(); i++) {
            if (s.substring(i, i+1).equals(" ")) {
                answer += " ";
                flag = true;
            } else {
                if (flag) {
                    answer += s.substring(i, i+1).toUpperCase();
                    flag = false;
                } else {
                    answer += s.substring(i, i+1).toLowerCase();
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    flag = True;
    for i in s:
        if i == " ":
            flag = True;
            answer = answer + i
            continue;
        if flag:
            answer = answer + i.upper()
        else:
            answer = answer + i.lower()
        flag = False
    # s = s.strip()
    # ss = s.split()
    # print(ss)
    # for i in ss:
    #     if (len(i) > 0):
    #         answer = answer + i[0].upper() + i[1:].lower() + " "
    #     else:
    #         answer = answer + i[0].upper() + " "
    # answer = answer.strip()
    return answer